index.html

```diff
@@ -45,6 +45,155 @@ <h3 class="contact">
+            <h1 class="card-head">Multi-Post <br>Stories</h1>
+            <ul class="card-intro-list">
+                <li class="card-list-first-child card-list-item">CANOPY</li>
+                <li class="card-list-item">Back End Dev</li>
+                <li class="card-list-item">2015</li>
+            </ul>
+
+            <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+            <ul class="card-lang-list">
+                <li class="card-lang-list-item">html</li>
+                <li class="card-lang-list-item">css</li>
+                <li class="card-lang-list-item">JavaScript</li>
+            </ul>
+
+            <a class="card-button-link" href="#" rel="noopener noreferrer" target="_blank">See Project</a>
+        </div>
+
+        <div class="work-card">
+            <img class="card-image" src="images/work-card3.svg" alt="project-3">
+            <h1 class="card-head">Blog</h1>
+            <ul class="card-intro-list">
+                <li class="card-list-first-child card-list-item">CANOPY</li>
+                <li class="card-list-item">Front End Dev</li>
+                <li class="card-list-item">2019</li>
+            </ul>
+
+            <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+            <ul class="card-lang-list">
+                <li class="card-lang-list-item">html</li>
+                <li class="card-lang-list-item">css</li>
+                <li class="card-lang-list-item">JavaScript</li>
+            </ul>
+
+            <a class="card-button-link" href="#" rel="noopener noreferrer" target="_blank">See Project</a>
+        </div>
+
+        <div class="work-card">
+            <img class="card-image" src="images/work-card4.svg" alt="project-4">
+            <h1 class="card-head">Multi-Post <br>Stories</h1>
+            <ul class="card-intro-list">
+                <li class="card-list-first-child card-list-item">CANOPY</li>
+                <li class="card-list-item">Front End Dev</li>
+                <li class="card-list-item">2020</li>
+            </ul>
+
+            <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+            <ul class="card-lang-list">
+                <li class="card-lang-list-item">html</li>
+                <li class="card-lang-list-item">css</li>
+                <li class="card-lang-list-item">JavaScript</li>
+            </ul>
+
+            <a class="card-button-link" href="#" rel="noopener noreferrer" target="_blank">See Project</a>
+        </div>
+    </section>
+
+    <section class="about-myself">
+        <h1 class="about-title">About <br>Myself</h1>
```

Comment:
As mentioned earlier regarding `h1` tags applies here as well.

- [ ] Please replace this `h1` tag with another one (maybe `h2` tag).